src/Panel/Document.php

```diff
@@ -158,22 +147,59 @@ public static function customCss(): ?string
     }
 
     /**
-     * Check for a custom js file from the
-     * config (panel.js)
-     *
-     * @return string|null
+     * @deprecated 3.7.0 Use `Document::customAsset('panel.css)` instead
+     * @todo add deprecation warning in 3.7.0, remove in 3.8.0
+     */
+    public static function customCss(): ?string
+    {
+        return static::customAsset('panel.css');
+    }
+
+    /**
+     * @deprecated 3.7.0 Use `Document::customAsset('panel.js)` instead
+     * @todo add deprecation warning in 3.7.0, remove in 3.8.0
      */
     public static function customJs(): ?string
     {
-        if ($js = kirby()->option('panel.js')) {
-            $asset = asset($js);
+        return static::customAsset('panel.js');
+    }
 
-            if ($asset->exists() === true) {
-                return $asset->url() . '?' . $asset->modified();
-            }
+    /**
+     * Returns array of favion icons
+     * based on config option
+     * @since 3.6.2
+     *
+     * @param string $url URL prefix for default icons
+     * @return array
+     */
+    public static function favicon(string $url = ''): array
+    {
+        $icons = kirby()->option('panel.favicon', [
+            'apple-touch-icon' => [
+                'type' => 'image/png',
+                'url'  => $url . '/apple-touch-icon.png',
+            ],
+            'shortcut icon' => [
+                'type' => 'image/svg+xml',
+                'url'  => $url . '/favicon.svg',
+            ],
+            'alternate icon' => [
+                'type' => 'image/png',
+                'url'  => $url . '/favicon.png',
+            ]
+        ]);
+
+        // make sure to convert favicon string to array
+        if (is_string($icons) === true) {
+            $icons = [
+                'shortcut icon' => [
+                    'type' => 'image/svg+xml',
```

Comment:
Does it have to be the default SVG? What if we set the mime value dynamically?

````php
'type' => F::mime($icons),
````

Reply:
True.... but I think then we need an absolute path for `$icons`, right?

Reply:
I'm not sure but yes. We can prefix kirby root path:

````php
'type' => F::mime($kirby->root('index') . '/' . ltrim($icons, '/')),
````

Reply:
Would 

```php
return [
  'panel' => [
    'favicon' => 'assets/favicon.ico'
  ]
];
```

or do we always need an absolute path?

Reply:
I'm really not sure of that either. Because it may not work without absolute path in multiple or custom domain setups? May be @lukasbestle have an idea about that.

Reply:
I just pushed a new version that I think works